utilities/transactions/transaction_util.cc

```diff
@@ -56,6 +56,12 @@ Status TransactionUtil::CheckKey(DBImpl* db_impl, SuperVersion* sv,
                                  const std::string& key, bool cache_only,
                                  ReadCallback* snap_checker,
                                  SequenceNumber min_uncommitted) {
+  // When `min_uncommitted` is provided, keys are not always committed
+  // in sequence number order, and`snap_checker` is used to check whether
```

Comment:
minor: space between and `snap_checker`
minor: I do not recall we using ` in the comments

Reply:
I don't remember any clear rule about "`" either, but there were already some "`" in the code base since I joined the team. Some people use it and some don't. I can remove "`" if you don't like it. It's not a big deal.

Reply:
not a big deal.